src/workers/sync-engine/modules/folders/domain/FolderPath.ts

```diff
@@ -17,7 +17,7 @@ export class FolderPath extends Path {
       return 'Internxt Drive';
     }
 
-    return super.name();
+    return super.name(true);
```

Comment:
If the logic is not the same for all Paths (files and folders) I would rather not call the parent class and return here `path.basename(this.value)` as we already know it's folder because we are in a FolderPath method. Nevertheless, looks good to me to merge